app/assets/javascripts/modules/paste-html-to-govspeak.js

```diff
@@ -0,0 +1,11 @@
+//= require paste-html-to-govspeak/dist/paste-html-to-markdown.js
+
+(function (Modules) {
+  'use strict'
+
+  Modules.PasteHtmlToGovspeak = function () {
+    this.start = function (element) {
+      element[0].addEventListener('paste', window.pasteHtmlToGovspeak.pasteListener)
+    }
+  }
+})(window.GOVUKAdmin.Modules)
```

Comment:
It doesn't look like this app makes use of the rather legacy GOVUKAdmin.Modules system already so I don't think this is something we want to introduce here - I imagine it just couples us into the legacy govuk_admin_template. I'd imagine it'd be done neater with just a jQuery initialiser as you had above for now and then use regular window.GOVUK.Modules once govuk_publishing_components are installed.